extraConfigDpuTenant.py

```diff
@@ -147,18 +162,17 @@ def run(self, cfg, futures: Dict[str, Future]) -> None:
         iclient.oc(f"project tenantcluster-dpu")
         print(iclient.oc(f"create secret generic tenant-cluster-1-kubeconf --from-file=config={tclient._kc}"))
 
-        contents = open("manifests/tenant/envoverrides.yaml").read()
-        for e in cfg["mapping"]:
-            a = {}
-            a["TENANT_K8S_NODE"] = e['worker']
-            a["DPU_IP"] = iclient.get_ip(e['bf'])
-            a["MGMT_IFNAME"] = "c1pf0vf0"
-            contents += f"  {e['bf']}: |\n"
-            for (k, v) in a.items():
-                contents += f"    {k}={v}\n"
-        open("/tmp/envoverrides.yaml", "w").write(contents)
+        tc_namespace = "tenantcluster-dpu"
+        dpu_namespace = "openshift-dpu-network-operator"
+        file = self.render_envoverrides_cm(iclient, cfg, tc_namespace)
+        iclient.oc(f"create -f {file}")
+        file = self.render_envoverrides_cm(iclient, cfg, dpu_namespace)
+        iclient.oc(f"create -f {file}")
+
+        # Restart DPU network operator to apply env-overrides cm
+        iclient.oc("scale --replicas=0 deploy/dpu-network-operator-controller-manager -n openshift-dpu-network-operator")
```

Comment:
Don't wee need to wait for the scale down to apply completely (i.e. make sure that all pods stop) and then scale up only after that?

Reply:
Aahh right thanks. I was under the impression this method would work from when we tried it the other day, but I am unable to apply the cm this way on a new cluster (even after waiting for all pods to scale down). Updated the PR to restart w/ make undeploy instead